### PR TITLE
Changing file names, class names, function names

### PR DESCRIPTION
**Pull request by samwalkow**
I changed several file, classes and function names to match python naming conventions. I hopefully changed all references to the files and objects I changed, but I may have missed some.

### CODE REVIEW

**Comment by samwalkow** [comment]
Oops, forgot to adjust the tests, working on it!

**Comment by samwalkow** [comment]
I'm not sure how to get both the black and isort checks to agree? they seem to want different things?

**Comment by chrishavlin** [comment]
ah, that's annoying. There's probably a way that I can improve the pre-commit so that the black-isort loop doesn't conflict... but for now you can add a comment to the line that tells isort to skip that line: lines that end in ` # skip:isort` will get skipped (see https://pycqa.github.io/isort/docs/configuration/action_comments.html for more info)

File: analysis_schema/base_model_functions.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from pydantic import BaseModel
 
 
-def show_plots(schema, files):
+def ShowPlots(schema, files):
```
Comment (chrishavlin) [inline analysis_schema/base_model_functions.py:7]: this one's a function, so should stay `show_plots`, ya?

File: analysis_schema/data_classes.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 from pydantic import BaseModel, Field
 
-from .BaseModelFunctions import ytBaseModel, ytDataObjectAbstract, ytParameter
+from .base_model_functions import ytBaseModel, ytDataObjectAbstract, ytParameter
```
Comment (chrishavlin) [inline analysis_schema/data_classes.py:6]: `# skip: isort` should go here I think

**Comment by chrishavlin** [comment]
IMO, we should just merge this! I can fix all the linting in a subsequent PR, I think pre-commit vs the github action needs some work.

**Comment by chrishavlin** [comment]
Oh actually, i **think** I can fix the isort/black loop here

File: analysis_schema/data_classes.py [hunk 3]
```diff
@@ -3,9 +3,10 @@
...
```
Comment (chrishavlin) [inline analysis_schema/data_classes.py:6]: Ok, I was wrong, I think we need a split, not a skip!

```suggestion
from .base_model_functions import ytBaseModel, ytDataObjectAbstract, ytParameter  # isort: split
```

Comment (chrishavlin) [reply analysis_schema/data_classes.py:6]: I'm just going to go ahead and commit it...

**Comment by chrishavlin** [comment]
oops. isort passes, but it triggered a flake8 issue... I'll fix this on monday!! and then make black and isort play together in a more manageable way...

**Comment by chrishavlin** [comment]
actually gave it one more shot! Should be good to merge now assuming the python tests pass

**Comment by chrishavlin** [comment]
Ok,I'm going to merge this now. Thanks for improving the naming conventions @samwalkow !

**Comment by samwalkow** [comment]
@chrishavlin thank you for dealing with this! Really helpful!

**Pull request merged**